Makefile

```diff
@@ -414,27 +415,26 @@ flake8_collection:
 
 test_collection_all: prepare_collection_venv test_collection flake8_collection
 
-test_collection_sanity:
-	rm -rf sanity
-	mkdir -p sanity/ansible_collections/$(COLLECTION_NAMESPACE)
-	cp -Ra awx_collection sanity/ansible_collections/$(COLLECTION_NAMESPACE)/awx  # symlinks do not work
-	cd sanity/ansible_collections/$(COLLECTION_NAMESPACE)/awx && git init && git add . # requires both this file structure and a git repo, so there you go
-	cd sanity/ansible_collections/$(COLLECTION_NAMESPACE)/awx && ansible-test sanity
-
 # WARNING: symlinking a collection is fundamentally unstable
 # this is for rapid development iteration with playbooks, do not use with other test targets
 symlink_collection:
```

Comment:
I'm not sure we need this target at all anymore now that we have the sanity tests using the installed collection. We might just want to remove it, unless you think it provides us other value

Reply:
Without it, there is very little you can test locally without installing the collection. It takes 10 seconds to install on my machine (don't have to do it in the container). But the flow of doing a `cd` into `~/.ansible/collections/ansible_collections/awx/awx` sucks, because now you're running with a frozen, stale, copy of the code, and you're not in source control. Even if you used a 2nd terminal tab, that won't work, because the directory gets deleted on every install. This really needs improvement on the Ansible core side. If there was a better way, I would be with you, but I want to keep this as long as it's the best option for the purpose.